recipe/recipe.yaml

```diff
@@ -0,0 +1,90 @@
+context:
+  name: eigenpy
+  version: 3.10.0
+  build_num: 3
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/stack-of-tasks/${{ name }}/releases/download/v${{ version }}/${{ name }}-${{ version }}.tar.gz
+  sha256: 041ca892a9dab2cd81ba828aeb247adfd44438db5d6037ed61fde1e833a3edbe
+  patches:
+    - patches/0001-core-Allow-cxx11-compatibility.patch
+
+build:
+  number: ${{ build_num }}
+
+requirements:
+  build:
+    - ${{ compiler('cxx') }}
+    - ${{ stdlib("c") }}
+    - if: build_platform != target_platform
+      then:
+        - python
+        - cross-python_${{ target_platform }}
+        - numpy
+    - cmake
+    - clang
```

Comment:
I don't think we need clang. Compiler should be installed by `compiler('cxx')`.

Reply:
ok, clang is mandatory on windows because of clang-cl

Reply:
My bad, actually the `win` selector was missing for clang here